packages/extensions/openapi-to-typespec/test/arm-apimanagement/tsp-output/ApiManagementServiceResource.tsp

```diff
@@ -26,18 +26,25 @@ model ApiManagementServiceResource
   >;
 
   /**
-   * ETag of the resource.
+   * SKU properties of the API Management service.
    */
-  @visibility("read")
-  etag?: string;
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "For backward compatibility"
+  sku: ApiManagementServiceSkuProperties;
 
   /**
-   * A list of availability zones denoting where the resource needs to come from.
+   * Managed service identity of the Api Management service.
    */
-  zones?: string[];
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "For backward compatibility"
+  identity?: ApiManagementServiceIdentity;
+
+  /**
+   * ETag of the resource.
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "For backward compatibility"
+  @visibility("read")
+  etag?: string;
```

Comment:
Need to confirm if it could be replaced by `EntityTagProperty`, since in the `EntityTagProperty`, the name is `eTag`. Capital "T".

Reply:
This will be fixed in the next PR too.